admin-dev/themes/new-theme/scss/components/_modal.scss

```diff
@@ -0,0 +1,7 @@
+.modal {
```

Comment:
Well every modal in migrated pages have this issue with z-index (because header toolbar z-index was increased to 1100 in a recent pull request).

There are really a lot of modals out there and all of them would be fixed with this PR:
* On the catalog->products page there are 6 specific modals (for each bulk action and some others).
* In all modules pages (module manager, module catalog, module alerts, module updates) there are modals for uploading a module and connecting to addons (so 8 modals in total).
* "Read more" link in modules catalog on every module opens a modal.
* Import step 2 page has a modal.
* **Every** migrated page with grid has "show sql query" button that opens a modal.
* There are more of them. :confused: 

Do you think it's risky to have this global change? It's only increasing the z-index of a modal (which is supposed to be on the very top of z axis).
Wdyt @matks?

Reply:
It's risky for sure, the question is whether we can take the risk or not.

Your arguments are good, however this is important so I'm going to ping @Quetzacoalt91 and @PierreRambaud for their opinions.

Reply:
It's just a z-index change and a modal needs to be in front of everything, it's ok for me.

Reply:
I think we need to change z-index of .main-header and .header-toolbar to 1000 only, It will solve the issue. Wdyt? @Quetzacoalt91

Reply:
I'm not the most appropriate person to ping. In my opinion I would avoid making changes on bootstrap elements as it brings a lot of side effects.

Reply:
@eternoendless :)